tutorials/katacoda/thanos/2-lts/step3.md

```diff
@@ -0,0 +1,118 @@
+# Step 3 - Installing Thanos Store
```

Comment:
Let's configure object store (minio) in this step 4

and then Step 4) change sidecar to uplaod to object store

then step 5) Install Thanos Store